pkgs/development/python-modules/robotframework-browser/default.nix

```diff
@@ -0,0 +1,125 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pythonOlder,
+  setuptools,
+  robotframework,
+  robotframework-pythonlibcore,
+  robotframework-assertion-engine,
+  grpcio,
+  overrides,
+  click,
+  seedir,
+  wrapt,
+  npmHooks,
+  nodejs,
+  fetchNpmDeps,
+  node-pre-gyp,
+  grpc-tools,
+  grpcio-tools, # needed to generate --grpc_python_out + imported
+  # For inv build
+  invoke, # cmake-like for python
+  mypy-protobuf,
+  robotstatuschecker,
+  pytest,
+  beautifulsoup4,
+  # To provide the binaries to the browsers + maintain the same version between playwright node
+  # and playwright browsers
+  playwright-driver,
+}:
+buildPythonPackage rec {
+  pname = "robotframework-browser";
+  version = "19.6.2";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
```

Comment:
This is not needed any longer

Reply:
What do you mean? Python 9 is not packaged anymore? In any case isn't it good to keep it for semantic reasons?

Reply:
I mean that this line can be removed. The `pythonOlder` argument too.